lms/static/scripts/frontend_apps/components/dashboard/DashboardApp.tsx

```diff
@@ -8,18 +7,15 @@ import { useFetch } from '../../utils/fetch';
 import StudentsActivityTable from './StudentsActivityTable';
 
 export default function DashboardApp() {
-  const { assignmentId } = useParams<{ assignmentId: string }>();
   const { dashboard, api } = useConfig(['dashboard', 'api']);
-  const assignment = {
-    title: dashboard.assignment.title,
-    id: assignmentId,
-  };
-  const students = useFetch<StudentStats[]>(`${assignmentId}_students`, () =>
-    apiCall({
-      authToken: api.authToken,
-      method: 'GET',
-      path: dashboard.assignmentStatsApi.path,
-    }),
+  const students = useFetch<StudentStats[]>(
+    `${api.authToken}_${dashboard.assignmentStatsApi.path}_students`,
```

Comment:
I forgot that there is a `useAPIFetch` wrapper around `useFetch`, that will handle determining the key itself.